src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PropertyPages/Editors/PropertyEditorBase.cs

```diff
@@ -13,20 +13,20 @@ namespace Microsoft.VisualStudio.ProjectSystem.VS.Implementation.PropertyPages.D
     /// </summary>
     internal abstract class PropertyEditorBase : IPropertyEditor
     {
-        private static readonly Lazy<ResourceDictionary> LazyResources;
-        private static readonly Lazy<DataTemplate> LazyGenericPropertyTemplate;
+        private static readonly Lazy<ResourceDictionary> s_lazyResources;
```

Comment:
>s_lazyResources [](http://example.com/codeflow?start=57&length=15)

This is all just naming convention changes in this file, correct?

Reply:
Yes, just naming conventions.